src/activities/photo_hunter/Observe.qml

```diff
@@ -0,0 +1,145 @@
+ */
+import QtQuick 2.1
+import GCompris 1.0
+
+import "../../core"
+import "photo_hunter.js" as Activity
+
+Image {
+    id: card
+
+    sourceSize.width: background.vert ? undefined : (background.width - 30) / 2
+    sourceSize.height: background.vert ? (background.height - background.barHeight - 40) /2 :
+                                         background.height - background.barHeight - 30
+
+    property GCAudio audioEffects: activity.audioEffects
+    property alias repeater: repeater
+    property int good: 0
+
+    Image {
+        id: wrong
+        source: Activity.url + "images/wrong.svg"
+        width: 70
+        height: 70
+        opacity: 0
+    }
+
+    NumberAnimation {
+        id: wrongAnim;
+        target: wrong;
+        property: "opacity";
+        from: 0; to: 1;
+        duration: 400
+    }
+
+    NumberAnimation {
+        id: wrongAnim2;
+        target: wrong;
+        property: "opacity";
+        from: 1; to: 0;
+        duration: 400
+    }
+
+    MouseArea {
+        id: big
+        anchors.fill: parent
+        onClicked: {
+            audioEffects.play('qrc:/gcompris/src/core/resource/sounds/brick.wav')
+            wrongAnim.start()
+            wrong.x = mouseX - wrong.width/2
+            wrong.y = mouseY - wrong.height/2
+            wrongAnim2.start()
+        }
+    }
+
+    Repeater {
+        id: repeater
+
+        model: items.model
+        Image {
+            id: photo
+            property alias particleLoader: particleLoader
+            property alias scaleAnim: scaleAnim
+            property double widthScale
+            property double heightScale
+
+            source: Activity.url + "images/circle.svg"
+            width: card.width / 10 * widthScale
+            height: card.height / 10 * heightScale
+
+            opacity: 1
+
+            x: modelData[0] * card.width / 1200
+            y: modelData[1] * card.height / 1700
+
+
+            //instead of scale, there should be an animation for both width and height
+            //-> so it will be able to draw an elipse if needed
+            NumberAnimation {
+                id: scaleAnim;
+                target: photo;
+                property: "scale";
+                from: 0; to: photo.scale;
+                duration: 700
+            }
+
+            // Create a particle only for the strawberry
+            Loader {
+                id: particleLoader
+                anchors.fill: parent
+                active: false
+                sourceComponent: particle
+            }
+
+            Component {
+                id: particle
+                ParticleSystemStarLoader
+                {
+                    id: particles
+                    clip: false
+                }
+            }
+
+            MouseArea {
+                id: mouseArea
+                anchors.fill: parent
+                onClicked: {
+                    img1.repeater.itemAt(index).particleLoader.active = active
```

Comment:
It would be better to call a function on the items instead of being intrusive here.